blog/2025-05-22-lw-registry/index.mdx

```diff
@@ -0,0 +1,146 @@
+---
+slug: distr-registry
+title: 'Introducing the Distr Artifact Registry'
+description: 'A built-in OCI registry with tag-based access control, audit logs, and support for Helm, Docker & more, designed to simplify self-managed software distribution for both vendors and customers.'
+authors: [jpage]
+tags: ['OCI Registry', 'distr-launch-week']
+image: 'TODO'
+---
+
+import DefaultCTA from '@site/src/components/cta/DefaultCTA/defaultCTA';
+import ThemedImage from '@theme/ThemedImage';
+
+Today, we are happy to announce that the Distr [artifact registry](https://distr.sh/docs/product/registry/) is now Generally Available (GA).
+
+Manage and distribute container images, Helm charts, and other OCI artifacts with license-based access control, audit logs, and a visual UI that makes self-managed software distribution pretty darn simple if we do say so ourselves, built for both you and your customers.
+
+<!-- truncate -->
+
+## Why we built a registry into Distr
```

Comment:
I would rephrase these to sections (https://github.com/glasskube/glasskube.dev/pull/124/files#diff-f2b2f931d3c280bc597cd4f758af93c7945cfaaafe471d1f88032407b4b6433cR18-R53)

to indicate that previously we only supported the assisted self manged use case, but due to end customer requirements, software vendors asked us to also support the fully self managed use case.

(it can also be truncated to one paragraph - it doesn't need to be 2 sections)